umbraco-cloud/set-up/project-settings/README.md

```diff
@@ -1,160 +1,218 @@
+* Monitor pending project invitations.
+
+<figure><img src="../../.gitbook/assets/image (59).png" alt=""><figcaption></figcaption></figure>
+
+### Summary
 
-### [Edit team](team-members/)
+The **Summary** section displays key information such as the project plan, the region where the project was created, payment status, and more.
 
-Manage the team members and user permissions on your project. You can also view the backoffice user groups for each team member, view pending project invites, and manage [Technical contacts](team-members/technical-contact.md) for your project.
+![Project Summary](../images/summary-details.png)
 
 ## Insights
 
 ### [Project History](project-history.md)
 
-On the project history page, you can see a history of activities that have happened on your projects.
+In the Project History section, you can view a list of high-level activities for your cloud project.
+
+![Project History](../images/project-history.png)
 
 ### [Availability & Performance](availability-performance.md)
 
-You can see metrics related to the overall health and performance of the Azure app service hosting the live environment of your solution.
+You can see metrics related to the overall health and performance of the Azure app service hosting the different environments of your solution.
+
+![Availability & Performance](../images/availability-performance.png)
 
 ### [Usage](usage.md)
 
-On your Umbraco Cloud project, it is possible to see the usage of Custom Domains, Media Storage, Content Nodes, and Bandwidth for the project. You can also check if it is using above or below the allowed amount for the plan that your project is on.
+The Usage section allows you to:
+
+* View the usage of Custom Domains, Media Storage, Bandwidth, and Bandwidth History for your project.
+* Check whether the project is using above or below the allowed amount for its plan.
+* View the top 10 bandwidth usage paths, referrers, and the top 50 media files.
+
+![Usage](../images/usage-new.png)
 
 ## Configuration
 
-### [Connection details](../../databases/)
+### [Connections](../../databases/)
 
-Find connection details to your Umbraco Cloud databases. You need to allow your IP to connect to the databases with your local machine.
+The Connections section provides connection details for your Umbraco Cloud databases. You need to allow your IP to connect to the databases with your local machine.
+
+![Connections](../images/connections.png)
 
 ### [Automatic Upgrades](../../product-upgrades/minor-upgrades.md)
 
-We handle minor and patch upgrades for the Umbraco components used by Umbraco Cloud, so you don't have to. From the Automatic Upgrades page, you can control if you want to opt in or out of automatic minor upgrades.
+The Automatic Upgrades section handles minor and patch upgrades for the Umbraco components used by Umbraco Cloud. By default, new projects are opt-in for these upgrades.
+From here, you can control whether you opt in or out of automatic minor upgrades.
```

Comment:
Should we make this even simpler?

Something like": "From this page, you can control whether your site is automatically upgraded to the latest minor version(s) of the Cloud products. Learn more about what upgrades happen automatically in the Upgrades section of the Cloud documentation." (and then link to the upgrades docs 🤔 )

Reply:
Makes sense. I updated the statement.